Source/SwiftLintFramework/Rules/UnusedClosureParameterRule.swift

```diff
@@ -93,6 +106,8 @@ public struct UnusedClosureParameterRule: ASTRule, ConfigurationProviderRule, Co
                 return nil
             }
 
+            let paramLength = name.characters.count
```

Comment:
you should use `name.bridge().length` instead as `name.characters.count` can return a different value